registrations/tasks.py

```diff
@@ -273,8 +274,10 @@ def create_subscriptionrequests(self, registration):
         weeks = 1  # default week number
 
         # . calculate weeks along
-        if (registration.reg_type == "momconnect_prebirth" and
-           registration.source.authority not in ["hw_partial", "patient"]):
+        if (registration.reg_type in [
+                "momconnect_prebirth", "whatsapp_prebirth"] and
+                registration.source.authority not in [
+                    "hw_partial", "patient"]):
```

Comment:
Yeah so perhaps I should leave it out entirely. I'll do that and see which tests fail, if any.